bot/persistence/database_connector.py

```diff
@@ -155,6 +155,105 @@ def remove_module_role(self, role_id: int):
+                return int(row[0])
+            return None
+
+    def add_reaction_role(self, msg_id: int, emoji: str, role_id: int):
+        """Adds information needed for a reaction role to the table "ReactionRole".
+
+        Args:
+            msg_id (int): The id of the message which users should react to.
+            emoji (str): The emoji for the reaction role.
+            role_id (int): The id of the role for the reaction role.
+        """
+        with DatabaseManager(self._db_file) as db_manager:
+            db_manager.execute(queries.INSERT_REACTION_ROLE, (msg_id, emoji, role_id))
+            db_manager.commit()
+
+    def remove_reaction_role(self, msg_id: int, emoji: str):
+        """Removes information needed for a reaction role from the table "ReactionRole".
+
+        Args:
+            msg_id (int): The id of the message which users should react to.
+            emoji (str): The emoji for the specific reaction role.
+        """
+        with DatabaseManager(self._db_file) as db_manager:
+            db_manager.execute(queries.REMOVE_REACTION_ROLE, (msg_id, emoji))
+            db_manager.commit()
+
+    def clear_reaction_roles(self, msg_id: int) -> bool:
+        """Removes all information needed for the reaction roles of a specific message from the table "ReactionRole".
+
+        Args:
+            msg_id (int): The id of the message which reaction roles should be removed.
+
+        Returns:
+            bool: A boolean indicating if any reaction roles have been deleted.
+        """
+        with DatabaseManager(self._db_file) as db_manager:
+            affected_rows = db_manager.execute(queries.CLEAR_REACTION_ROLES, (msg_id,)).rowcount
+            db_manager.commit()
+
+            return affected_rows != 0
+
+    def add_reaction_role_group(self, msg_id: int):
+        """Adds the id of a message to the table "ReactionRoleGroup".
+
+        The existence of a message id in this table indicates, that a user should only be able to have one of the
+        specified reaction roles of a message at any time given.
+
+        Args:
+            msg_id (int): The id of the message which users can react to.
+        """
+        with DatabaseManager(self._db_file) as db_manager:
+            db_manager.execute(queries.INSERT_REACTION_ROLE_GROUP, (msg_id,))
+            db_manager.commit()
+
+    def remove_reaction_role_group(self, msg_id: int):
+        """Removes the id of a message from the table "ReactionRoleGroup".
+
+        The absence of a message id in this table indicates, that a user can have multiple reaction roles of this
+        message at once.
+
+        Args:
+            msg_id (int): The id of the message which users can react to.
+        """
+        with DatabaseManager(self._db_file) as db_manager:
+            db_manager.execute(queries.REMOVE_REACTION_ROLE_GROUP, (msg_id,))
+            db_manager.commit()
+
+    def check_reaction_role_uniqueness(self, msg_id: int) -> bool:
```

Comment:
exclusiveness instead of uniqueness

Reply:
I think that exclusiveness isn't the right term here. Just because something is exclusive, doesn't mean that you can have only one of it. I find uniqueness more fitting to be honest.

Reply:
Well, I mean exclusiveness does exactly mean that it "excludes" all other options.